zmscitizenapi/src/Zmscitizenapi/Services/Core/ZmsApiFacadeService.php

```
@@ -44,6 +44,49 @@ private static function getError(string $key): array
+
+        $keys = [
+            self::CACHE_KEY_OFFICES,
+            self::CACHE_KEY_OFFICES . '_unpublished',
+            self::CACHE_KEY_SCOPES,
+            self::CACHE_KEY_SERVICES,
+            self::CACHE_KEY_SERVICES . '_unpublished',
+            self::CACHE_KEY_OFFICES_AND_SERVICES,
+            self::CACHE_KEY_OFFICES_AND_SERVICES . '_unpublished',
+        ];
+
+        $sourceName = \App::$source_name ?? 'dldb';
```

PHPStan (2.2.7)

[error] Access to an undefined static property App::$source_name.


(staticProperty.notFound)